params_test.go

```diff
@@ -20,3 +20,67 @@ func TestFormatTime(test *testing.T) {
 	formattedTime := formatTime(time)
 	t.Assert(formattedTime, formatTime(time), "formatTime()")
 }
+
```

Comment:
should we uncomment these so they're being tested?

Reply:
The code moved to `request_options.go`, but I haven't ported all of the tests over. I just commented these so that I can reference them later. They will be removed when ported over